### PR TITLE
Add .rosinstall file for melodic fetch user

### PR DESCRIPTION
**Pull request by 708yamaguchi**
In this pull request, I add .rosinstall file for melodic fetch user.

In melodic, many fetchrobotics packages are released in apt, so we can simplify .rosinstall file compared to kinetic one.
kinetic version .rosinstall: https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_fetch_robot/jsk_fetch_user.rosinstall.kinetic

On my melodic Note PC, I checked that 
- [x] I can build catkin workspace by following the steps in the updated README.md (I do not execute wstool merge command because jsk_fetch_user.rosinstall.melodic is not online yet)
- [x] I can send angle-vector to real fetch robot via fetcheus. (send *ri* :angle-vector (send *fetch* :reset-pose) 10000)
- [x] I can move real fetch robot to other positions via fetcheus. (I used go-to-spot() function in jsk_fetch_startup/euslisp/navigation-utils.l)
- [x] I can send speech text to real fetch robot. (send \*ri* :speak-jp "こんにちは。")
- [x] I can receive speech text from real fetch robot. (rostopic echo /speech_to_text_google)

### CODE REVIEW

File: jsk_fetch_robot/jsk_fetch_user.rosinstall.melodic [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This file is for work space in indigo user PC.
```
Comment (k-okada) [inline jsk_fetch_robot/jsk_fetch_user.rosinstall.melodic:1]: indigo -> melodic?

File: jsk_fetch_robot/README.md [hunk 2]
```diff
@@ -47,6 +47,8 @@ if [ $ROS_DISTRO = "indigo" ]; then
   wstool merge -t . https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_fetch_robot/jsk_fetch_user.rosinstall.indigo
 elif [ $ROS_DISTRO = "kinetic" ]; then
   wstool merge -t . https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_fetch_robot/jsk_fetch_user.rosinstall.kinetic
+elif [ $ROS_DISTRO = "melodic" ]; then
+  wstool merge -t . https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_fetch_robot/jsk_fetch_user.rosinstall.melodic
```
Comment (k-okada) [inline jsk_fetch_robot/README.md:51]: how about 
```
if [[ "$ROS_DISTRO" =~ ^(indigo|kinetic|meldoic)$ ]]; then
wstool merge -t . http......jsk_fetch_user.rosinstall.$ROS_DISTRO`
else
echo "..."
fi
```

File: jsk_fetch_robot/jsk_fetch_user.rosinstall.melodic (hunk 1 above)
Comment (708yamaguchi) [reply jsk_fetch_robot/jsk_fetch_user.rosinstall.melodic]: Thank you for your review.
I fixed the typo.

File: jsk_fetch_robot/README.md (hunk 2 above)
Comment (708yamaguchi) [reply jsk_fetch_robot/README.md]: Thank you for your advice.
I updated the if-statement like your style.

**Pull request merged**